server/plugin/api.go

```diff
@@ -484,6 +484,7 @@ func (p *Plugin) completeConnectUserToGitHub(c *Context, w http.ResponseWriter,
 			"github_client_id":    config.GitHubOAuthClientID,
 			"enterprise_base_url": config.EnterpriseBaseURL,
 			"organization":        config.GitHubOrg,
+			"plugin_settings":     config.ClientSafeSettings(),
```

Comment:
This ensure the redux state is updated and the client doesn't need to call `/connected` again